src/components/Calendar/EventModal.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import { CalendarEvent } from "@/components/Calendar/Calendar";
+import styled from "styled-components";
```

Comment:
I'm just curious but can we use styled-components in a file without specifying `use-client`?

Reply:
We are not allowed
StyledComponent is a client-side thing

Reply:
Can you add it in then?

Reply:
Done